### PR TITLE
Johnson ogwuru calculator

### PR DESCRIPTION
**Pull request by ogwurujohnson**

### CODE REVIEW

File: Project-II-Calculator/calculator/src/components/ButtonComponents/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            
+            {/* loop through to getvall numbers passed through a state in the main container file */}
+            <div className="actionNumber">
+                <ActionButton text={clear} actionStyle={style.actionButton}/>
+                
+                <div className="buttonSection">
+                    {number.map(numberButton => {
+                        console.log(numberButton);
+                        return <NumberButton key={numberButton} text={numberButton} style={style.numberButton}/>
+                    })}
```
Comment (mattshardman) [inline Project-II-Calculator/calculator/src/components/ButtonComponents/index.js]: God job using ```.map```

File: Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js [hunk 2]
```diff
@@ -1,2 +1,11 @@
-import React from 'react';
+import React, {Fragment} from 'react';
 import './Button.css';
+
+export default function NumberButton(props) {
+    const {style, text} = props;
+    return (
+        <Fragment>
```
Comment (mattshardman) [inline Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js]: You can use ```<> </>``` as shorthand for Fragment (although you can't add keys and some tools don't support it) - just FYI!

File: Project-II-Calculator/calculator/src/container/calculator/index.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+export default class index extends Component {
+  state = {
+    operationStyle: "",
+    sum: 0,
+    clear: "Clear",
+    zero: 0,
+    number: [7,8,9,4,5,6,1,2,3],
+    operators: ["÷","x","-","+","="],
+  }
```
Comment (mattshardman) [inline Project-II-Calculator/calculator/src/container/calculator/index.js]: I probably won't add things that don't need to change into the state object. If they don't need to change they can just be declared as separate variables. State should be reserved for things that update.

**Comment by mattshardman** [comment]
Great job 

- [x]  Your calculator project closes resembles the design files
- [x]  Your git work flow is fluid and descriptive
- [x]  Your have separated the buttons and screen into separate components logically
- [x]  Good job using .map to keep your code DRY

**Possible things to work on**
- Make sure you are using state for things that will change within the application

Overall good job, everything is implemented correctly. and you calculator looks nice!

File: Project-II-Calculator/calculator/src/container/calculator/index.js (hunk 3 above)
Comment (ogwurujohnson) [reply Project-II-Calculator/calculator/src/container/calculator/index.js]: Taken note of, thanks

**Pull request merged**